.bit/responses/3.2-Week 3 Step 2.md

```diff
@@ -56,17 +58,21 @@ To test your work, you'll be using Postman to send a POST request in Postman wit
 <summary>❓ How do you check your blob storage container to see if the image is stored there?</summary>
 </br>
 
-![https://user-images.githubusercontent.com/69332964/99189316-9c592980-272e-11eb-9870-dbc1f9352599.png](https://user-images.githubusercontent.com/69332964/99189316-9c592980-272e-11eb-9870-dbc1f9352599.png)
+![image](https://user-images.githubusercontent.com/69332964/180457592-2d380233-f691-4b32-952f-7aafbf7f8fc1.png)
 
 <br><br>
 </details>
 
 ## Writing our *First* Azure Function to Upload an Image
 
+> ❗️ **Note:** In this project, we will be using the approach of **SDKs** and **API calls** to connect our Blob Storage and Azure Function. Another way to do this is through [binding](https://docs.microsoft.com/azure/azure-functions/functions-triggers-bindings?tabs=csharp), which is often faster but is sometimes limited.
```

Comment:
Instead of faster (which might be misunderstood with regards to performance or throughput I would state easier or more convenient